css-layout-classes/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header>
+            <h1>Choose your character!</h1>
+          </header>
+        </div>
+      </div>
+
+      <div class="row">
+        <article>
+          <div class="row">
```

Comment:
You won't ever really have `row`'s directly inside of another `row`. It will always go `row` > `column`

So you should delete the outter `'row'` and `article` and just start with the `<div>` on line 21